docs/en_US/TrainingService/PaiMode.md

```diff
@@ -92,4 +87,6 @@ Check policy:
 3. Note that the version check feature only check first two digits of version.For example, NNIManager v0.6.1 could use trialKeeper v0.6 or trialKeeper v0.6.2, but could not use trialKeeper v0.5.1 or trialKeeper v0.7.
 
 If you could not run your experiment and want to know if it is caused by version check, you could check your webUI, and there will be an error message about version check.
-![](../../img/version_check.png)
\ No newline at end of file
+![](../../img/version_check.png)
+
+Any problems when using NNI in pai mode, please create issues on [NNI github repo](https://github.com/Microsoft/nni).
```

Comment:
Suggest to remove this sentence,  the process to create issue is identical for all nni component,  no difference.

Reply:
fixed.